### PR TITLE
Update clojure-lsp repo and version

### PR DESCRIPTION
**Pull request by mracos**

### CODE REVIEW

File: installer/install-clojure-lsp.cmd [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @echo off
 
 setlocal
-set VERSION=20200706T152722
```
Comment (mracos) [inline installer/install-clojure-lsp.cmd:4]: I'm not that well versed with windows' cmd, so I'm not sure if this is right 😄

Comment (mattn) [reply installer/install-clojure-lsp.cmd:4]: Thanks. But please remove double-quotes.

Comment (mracos) [reply installer/install-clojure-lsp.cmd:4]: [done here](https://github.com/mattn/vim-lsp-settings/pull/385/commits/de33e9544979f4a707915964bdc09cd87fc67b5d)!

**Comment by mattn** [comment]
Thank you!

**Pull request merged**